apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -728,6 +728,10 @@ static void utc_audio_pcm_drop_p(void)
 
 	while (remain > 0) {
 		frames_read = pcm_readi(g_pcm, buffer, pcm_get_buffer_size(g_pcm));
+		if (frames_read == -EPIPE) {
+			pcm_prepare(g_pcm);
+			continue;
+		}
 		if (frames_read < 0) {
```

Comment:
Are you sure  it will succeed in reading / writing the second time.  There should be a way to check prepare status and try again.
Suggestion:
Retry mechanism has to be modified  based on some timeout mechanism. Or any better idea if you have

Reply:
Fixed. 
Added retry mechanism